packages/terra-application/src/primary-navigation-layout/PrimaryNavigationLayout.jsx

```diff
@@ -145,6 +146,10 @@ const propTypes = {
    * breakpoints.
    */
   utilityItems: utilityItemsPropType,
+  /**
+   * A PrimaryNavigationWorkspace element composed of PrimaryNavigationWorkspaceItem(s).
+   */
+  workspace: PropTypes.element,
```

Comment:
With the Page component, I started throwing exceptions if an unsupported sub-component was provided. We should consider doing the same here for the Workspace component: https://github.com/cerner/terra-application/blob/terra-application-v2/packages/terra-application/src/page/Page.jsx#L71-L77

Reply:
Updated.